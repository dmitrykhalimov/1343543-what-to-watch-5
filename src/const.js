export const ratingRanks = [
  {
    'Bad': 2.9
  },
  {
    'Normal': 4.9
  },
  {
    'Good': 7.9
  },
  {
    'Very good': 9.9
  },
  {
    'Awesome': 10
  }
];
