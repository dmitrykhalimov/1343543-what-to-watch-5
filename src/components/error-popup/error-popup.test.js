import React from "react";
import renderer from "react-test-renderer";
import ErrorPopup from "./error-popup";
import {TEST_MOCKS} from "../../const";

describe(`ErrorPopup`, () => {
  it(`Should ErrorPopup render correctly`, () => {
    const tree = renderer
      .create(
          <ErrorPopup
            errorMessage={``}
            onCloseButtonClick={TEST_MOCKS.noop}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

