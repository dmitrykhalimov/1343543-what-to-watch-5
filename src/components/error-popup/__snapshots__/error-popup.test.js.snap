// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPopup Should ErrorPopup render correctly 1`] = `
<div
  style={
    Object {
      "background": "rgba(255,255,255, 0.7)",
      "height": "100vh",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "100vw",
      "zIndex": "100",
    }
  }
>
  <div
    style={
      Object {
        "background": "#150202",
        "height": "300px",
        "margin": "0 auto",
        "marginTop": "15vw",
        "paddingTop": "50px",
        "textAlign": "center",
        "width": "1000px",
      }
    }
  >
    <p
      style={
        Object {
          "color": "#DFCF77",
          "fontSize": "35px",
          "lineHeight": "50px",
        }
      }
    >
      
    </p>
    <button
      onClick={[Function]}
      style={
        Object {
          "background": "#DFCF77",
          "border": "none",
          "cursor": "pointer",
          "height": "50px",
          "marginTop": "20px",
          "width": "120px",
        }
      }
    >
      Закрыть
    </button>
  </div>
</div>
`;
